clusters/zero-trust/README.md

```diff
@@ -0,0 +1,65 @@
+# Zero-trust background
```

Comment:
@frodesundby or @jhrv do you remember why this was moved out from the doc in the first place?

Reply:
I believe we removed the background info because we decided that documentation and information like this didn't belong in our user documentation.

Reply:
Fair enough, but the question then becomes why the "Zero trust" references remain. Should they maybe be removed?